ui/src/tempVars/components/TemplatePreviewList.tsx

```diff
@@ -4,8 +4,8 @@ import uuid from 'uuid'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 import FancyScrollbar from 'src/shared/components/FancyScrollbar'
 
-const LI_HEIGHT = 35
-const LI_MARGIN_BOTTOM = 3
+const LI_HEIGHT = 28
```

Comment:
should these constants be in a separate constants file?

Reply:
I am going to refactor this later to not need constants